shared/src/sector/registered_proof.rs

```diff
@@ -369,21 +333,11 @@ macro_rules! i64_conversion {
 
 i64_conversion! {
     RegisteredPoStProof;
-    StackedDRGWinning2KiBV1 => 0,
-    StackedDRGWinning8MiBV1 => 1,
-    StackedDRGWinning512MiBV1 => 2,
-    StackedDRGWinning32GiBV1 => 3,
-    StackedDRGWinning64GiBV1 => 4,
-    StackedDRGWindow2KiBV1 => 5,
-    StackedDRGWindow8MiBV1 => 6,
-    StackedDRGWindow512MiBV1 => 7,
-    StackedDRGWindow32GiBV1 => 8,
-    StackedDRGWindow64GiBV1 => 9,
-    StackedDRGWindow2KiBV1P1 => 10,
-    StackedDRGWindow8MiBV1P1 => 11,
-    StackedDRGWindow512MiBV1P1 => 12,
-    StackedDRGWindow32GiBV1P1 => 13,
-    StackedDRGWindow64GiBV1P1 => 14,
+    StackedDRGWindow2KiBV1P1 => 0,
+    StackedDRGWindow8MiBV1P1 => 1,
+    StackedDRGWindow512MiBV1P1 => 2,
+    StackedDRGWindow32GiBV1P1 => 3,
+    StackedDRGWindow64GiBV1P1 => 4,
```

Comment:
We should not be re-assigning numbers.

Reply:
Agree. This was more or less done so we can drop array size in builtin-actors repository.(`REGISTERED_POST_PROOF_VARIANTS`).

Reply:
Yeah, unfortunately we can't do that (see [here](https://github.com/filecoin-project/builtin-actors/pull/1391#discussion_r1340149993)).